tests/lit/lit/formats/alive2test.py

```diff
@@ -49,9 +49,9 @@ class Alive2Test(TestFormat):
   def __init__(self):
     self.regex_errs = re.compile(r";\s*(ERROR:.*)")
     self.regex_xfail = re.compile(r";\s*XFAIL:\s*(.*)")
-    self.regex_args = re.compile(r";\s*TEST-ARGS:(.*)")
-    self.regex_check = re.compile(r";\s*CHECK:(.*)")
-    self.regex_check_not = re.compile(r";\s*CHECK-NOT:(.*)")
+    self.regex_args = re.compile(r"(;|//)\s*TEST-ARGS:(.*)")
```

Comment:
you can use non-capturing parentheses here to avoid all the changes below
e.g. `(?:;|//)